CalibCalorimetry/CaloMiscalibTools/plugins/HcalRecHitRecalib.cc

```diff
@@ -2,52 +2,45 @@
 
 #include "DataFormats/Common/interface/Handle.h"
 #include "DataFormats/HcalDetId/interface/HcalDetId.h"
-#include "Geometry/Records/interface/HcalRecNumberingRecord.h"
 #include "FWCore/ParameterSet/interface/FileInPath.h"
-#include "FWCore/Framework/interface/ESHandle.h"
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "CalibCalorimetry/CaloMiscalibTools/interface/MiscalibReaderFromXMLHcal.h"
 
-HcalRecHitRecalib::HcalRecHitRecalib(const edm::ParameterSet& iConfig) {
-  tok_hbhe_ = consumes<HBHERecHitCollection>(iConfig.getParameter<edm::InputTag>("hbheInput"));
-  tok_ho_ = consumes<HORecHitCollection>(iConfig.getParameter<edm::InputTag>("hoInput"));
-  tok_hf_ = consumes<HFRecHitCollection>(iConfig.getParameter<edm::InputTag>("hfInput"));
-
+HcalRecHitRecalib::HcalRecHitRecalib(const edm::ParameterSet& iConfig)
+    : tok_hbhe_(consumes<HBHERecHitCollection>(iConfig.getParameter<edm::InputTag>("hbheInput"))),
+      tok_ho_(consumes<HORecHitCollection>(iConfig.getParameter<edm::InputTag>("hoInput"))),
+      tok_hf_(consumes<HFRecHitCollection>(iConfig.getParameter<edm::InputTag>("hfInput"))),
+      topologyToken_(esConsumes<edm::Transition::BeginRun>()),
+      RecalibHBHEHits_(iConfig.getParameter<std::string>("RecalibHBHEHitCollection")),
+      RecalibHFHits_(iConfig.getParameter<std::string>("RecalibHFHitCollection")),
+      RecalibHOHits_(iConfig.getParameter<std::string>("RecalibHOHitCollection")),
+      hcalfileinpath_(iConfig.getUntrackedParameter<std::string>("fileNameHcal", "")),
+      refactor_(iConfig.getUntrackedParameter<double>("Refactor", (double)1)),
+      refactor_mean_(iConfig.getUntrackedParameter<double>("Refactor_mean", (double)1)) {
   //   HBHEHitsProducer_ = iConfig.getParameter< std::string > ("HBHERecHitsProducer");
   //   HOHitsProducer_ = iConfig.getParameter< std::string > ("HERecHitsProducer");
   //   HFHitsProducer_ = iConfig.getParameter< std::string > ("HERecHitsProducer");
   //   HBHEHits_ = iConfig.getParameter< std::string > ("HBHEHitCollection");
   //   HFHits_ = iConfig.getParameter< std::string > ("HFHitCollection");
   //   HOHits_ = iConfig.getParameter< std::string > ("HOHitCollection");
```

Comment:
it is preferred to remove commented out code lines, or please mention why these might be needed in the future

Reply:
This is HCAL code that I migrated since it was practically the same than the ECAL one. However, I do not know if these comments have any value for the HCAL group and so I'd rather not touch them.

Reply:
@cms-sw/hcal-dpg-l2 can you please comment?

Reply:
Hi, this looks deprecated to me (and has not been touched in 8 years). There are corresponding commented lines further below that could be removed as well: https://github.com/cms-sw/cmssw/blob/e9138f1572558fc7857cbd17b6ce56273b750035/CalibCalorimetry/CaloMiscalibTools/plugins/HcalRecHitRecalib.cc#L85-L92